payments-core/src/main/java/com/stripe/android/payments/core/authentication/Stripe3ds2TransactionActivity.kt

```diff
@@ -0,0 +1,43 @@
+package com.stripe.android.payments.core.authentication
+
+import android.app.Activity
+import android.content.Intent
+import android.os.Bundle
+import androidx.activity.viewModels
+import androidx.appcompat.app.AppCompatActivity
+import com.stripe.android.payments.PaymentFlowResult
+
+/**
```

Comment:
Now that you are on this, could you define all 3ds2 related authentication classes to `payments.core.authentication.threeds2`? Ideally these classes would live in a separate module so they could be created from reflection(and make it possible to to optionally support 3ds2 sdk)

Reply:
`Stripe3DS2Authenticator` and classes defined in this PR